app/views/items/_search_form.html.erb

```diff
@@ -1,8 +1,17 @@
 <%= search_form_for @q, html: { :class => 'd-flex mx-3' } do |f| %>
 
     <%= f.search_field :name_or_description_cont, class: 'form-control me-2', placeholder: t("navbar.search") %>
+    
+    <% unless params[:q].nil? %>
```

Comment:
Dieser Block existiert ja auch in _filter_form. Ich checke ehrlich gesagt nicht was er macht. Könnt ihr da noch einen kurzen Kommentag ran klatschen?

Reply:
Dieser Block holt sich alle ransack query parameter die schon in der url stehen und schreibt sie in `hidden fields`, damit sie beim submitten des forms wieder durchgereicht werden. Das ist leider nötig um mehrere (verschiedenartige) ransack queries zu kombinieren. Also zum Beispiel wenn der user erst in der Suchleiste was sucht, dann wird beim Filtern danach der Suchparameter in ein hidden field des Filterforms übernommen und dann beide queries wieder in die url geschrieben.